app/models/proxy.rb

```diff
@@ -476,8 +476,8 @@ def force_apicast_version
   delegate :default_production_endpoint, :default_staging_endpoint,
            to: :deployment_strategy, allow_nil: true
 
-  delegate :default_production_endpoint_apiap, :default_staging_endpoint_apiap,
-           to: :deployment_strategy_apiap, allow_nil: true
+  delegate :hosted_default_production_endpoint, :hosted_default_staging_endpoint,
+           to: :hosted_service_deployment_strategy, allow_nil: true
```

Comment:
https://github.com/3scale/porta/pull/2190/files#r519862581

Reply:
Alternatively, you can do:

```ruby
delegate :default_staging_endpoint, :default_production_endpoint,
  to: :hosted_service_deployment_strategy,
  prefix: 'hosted'
```

and in the view call `proxy.hosted_default_staging_endpoint` (instead of `proxy.hosted_service_deployment_strategy.default_staging_endpoint`).